src/components/Table/Table.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState, forwardRef } from 'react'
+import styled, { css } from '@xstyled/styled-components'
+import { variant } from '@xstyled/system'
+import PropTypes from 'prop-types'
+
+import TableHeader from './TableHeader'
+import TableRow from './TableRow'
+import TableBody from './TableBody'
+import TableData from './TableData'
+import Avatar from './Avatar'
+
+import Checkbox from '../Checkbox'
+
+const Table = ({ type, children }) => <Container type={type}>{children}</Container>
+
+const typeVariant = variant({
+  prop: 'type',
+  default: 'regular',
+  variants: {
+    regular: css`
+      box-shadow: 0px 2px 4px rgba(33, 33, 33, 0.2);
+    `,
+    quiet: css`
+      border: 0;
+    `
+  }
+})
+
+const Container = styled.table`
+  ${typeVariant}
+  border-radius: 4px;
+  background: ${({ theme }) => theme.white};
+
+  min-width: 328px;
+  max-width: 100%;
+  height: 100%;
+  border-collapse: collapse;
+  text-align: left;
+
+  td:first-of-type,
+  th:first-of-type {
+    p label {
+      margin-right: 11px;
+    }
+    p span {
+      margin-right: 8px;
+    }
+  }
+
+  tr:hover {
+    background: #f5f5f5;
+  }
+
+  td:last-child {
+    p {
+      text-align: center;
+      border-radius: 4px;
```

Comment:
Aqui tu pode usar a seleção que o xstyled tem, se tu der uma olhada no `Theme.js` vai ver que temos um array assim `const radii = [0, 2, 4, 8]` e o 4px seria a posição 2 do array, então tu pode passar assim:
```suggestion
      border-radius: 2;
```

E tu pode fazer isso para todos os casos do projeto, com margins e paddings utilizando o array `spaces`, temos os de fontSize tb. Isso ajuda a melhorar a escalabilidade de projeto.